packages/doenetml-worker-rust/lib-js-wasm-binding/src/math.ts

```diff
@@ -0,0 +1,521 @@
+        ["\u03B6", "\\zeta "], // 'ζ'
+        ["\u03B7", "\\eta "], // 'η'
+        ["\u0398", "\\Theta "], // 'Θ'
+        ["\u03F4", "\\Theta "], // 'ϴ'
+        ["\u03B8", "\\theta "], // 'θ'
+        ["\u1DBF", "\\theta "], // 'ᶿ'
+        ["\u03D1", "\\theta "], // 'ϑ'
+        ["\u03B9", "\\iota "], // 'ι'
+        ["\u03BA", "\\kappa "], // 'κ'
+        ["\u039B", "\\Lambda "], // 'Λ'
+        ["\u03BB", "\\lambda "], // 'λ'
+        ["\u03BC", "\\mu "], // 'μ'
+        ["\u00B5", "\\mu "], // 'µ' should this be micro?
+        ["\u03BD", "\\nu "], // 'ν'
+        ["\u039E", "\\Xi "], // 'Ξ'
+        ["\u03BE", "\\xi "], // 'ξ'
+        ["\u03A0", "\\Pi "], // 'Π'
+        ["\u03C0", "\\pi "], // 'π'
+        ["\u03D6", "\\pi "], // 'ϖ' should this be varpi?
+        ["\u03C1", "\\rho "], // 'ρ'
+        ["\u03F1", "\\rho "], // 'ϱ' should this be varrho?
+        ["\u03A3", "\\Sigma "], // 'Σ'
+        ["\u03C3", "\\sigma "], // 'σ'
+        ["\u03C2", "\\sigma "], // 'ς' should this be varsigma?
+        ["\u03C4", "\\tau "], // 'τ'
+        ["\u03A5", "\\Upsilon "], // 'Υ'
+        ["\u03C5", "\\upsilon "], // 'υ'
+        ["\u03A6", "\\Phi "], // 'Φ'
+        ["\u03C6", "\\phi "], // 'φ' should this be varphi?
+        ["\u03D5", "\\phi "], // 'ϕ'
+        ["\u03A8", "\\Psi "], // 'Ψ'
+        ["\u03C8", "\\psi "], // 'ψ'
+        ["\u03A9", "\\Omega "], // 'Ω'
+        ["\u03C9", "\\omega "], // 'ω'
+        ["\u2212", "-"], // minus sign
+        ["\u22C5", " \\cdot "], // dot operator
+        ["\u00B7", " \\cdot "], // middle dot
+        ["\u222A", " \\cup "], // ∪
+        ["\u2229", " \\cap "], // ∩
+        ["\u221E", " \\infty "], // ∞
+        ["\u2205", " \\emptyset "], // ∅
+        ["\u2032", "'"], // ′
+    ];
+
+    for (let sub of substitutions) {
+        latexString = latexString.replace(new RegExp(sub[0], "g"), sub[1]);
+    }
+
+    let startLdotsMatch = latexString.match(
+        /^(\\ )*(\\ldots|\.(\\ )*\.(\\ )*\.)(\\ )*(.*)$/,
+    );
+
+    if (startLdotsMatch) {
+        let afterLdots = startLdotsMatch[6];
+        if (afterLdots[0] !== ",") {
+            latexString = "\\ldots," + afterLdots;
+        } else {
+            latexString = "\\ldots" + afterLdots;
+        }
+    }
+
+    let endLdotsMatch = latexString.match(
+        /^(.*?)(\\ )*(\\ldots|\.(\\ )*\.(\\ )*\.)(\\ )*$/,
+    );
+
+    if (endLdotsMatch) {
+        let beforeLdots = endLdotsMatch[1];
+        if (beforeLdots[beforeLdots.length - 1] !== ",") {
+            latexString = beforeLdots + ",\\ldots";
+        } else {
+            latexString = beforeLdots + "\\ldots";
+        }
+    }
+
+    // replace [space]or/and/U[space]
+    // with latex
+    latexString = latexString.replace(/(\b|\\ )or(\b|\\ )/g, "$1\\lor$2");
+    latexString = latexString.replace(/(\b|\\ )and(\b|\\ )/g, "$1\\land$2");
+
+    if (unionFromU) {
+        latexString = latexString.replace(/(\b|\\ )U(\b|\\ )/g, "$1\\cup$2");
+    }
+
+    return latexString;
```

Comment:
I do have a whole library, `unified-latex` that does latex parsing. It might make the bundle size bigger, but it would certainly be more robust than regexes.